### DISCUSSION TITLE
Is TrueZen Ataraxis supposed to keep the statusline?

### DISCUSSION THREAD

**Discussion by ducnyn** [NvChad/NvChad#1340] [Q&A]
Hi, when use :TZAtaraxis or (space) ta the statusline jumps to the top and is shortened to the Ataraxis padding. Only when I press (space) again the statusline disappears. Is this default behavior?

I'm new to using nvim plugins and I know siduck said nvchad is not beginner friendly, but I just like the setup and found the config structure self-explanatory.  I mostly copied siduck's config but of course went through the code and only kept what I understood and want to use.

**Comment by siduck** [comment]
@dcngn its an upstream issue, ask it in truezen repo
